README.md

```diff
@@ -1,81 +1,115 @@
 # Azure Disk CSI driver for Kubernetes
+
 [![Travis](https://travis-ci.org/kubernetes-sigs/azuredisk-csi-driver.svg)](https://travis-ci.org/kubernetes-sigs/azuredisk-csi-driver)
 [![Coverage Status](https://coveralls.io/repos/github/kubernetes-sigs/azuredisk-csi-driver/badge.svg?branch=master)](https://coveralls.io/github/kubernetes-sigs/azuredisk-csi-driver?branch=master)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fkubernetes-sigs%2Fazuredisk-csi-driver.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fkubernetes-sigs%2Fazuredisk-csi-driver?ref=badge_shield)
 
 ### About
+
 This driver allows Kubernetes to use [Azure Disk](https://azure.microsoft.com/en-us/services/storage/disks/) volume, csi plugin name: `disk.csi.azure.com`
 
-### Project status: GA
+### Project status
+
+V1: GA
+
+V2: Preview
```

Comment:
Should we say Preview or Alpha?

Reply:
I believe Preview is fine here. I am using the Kubernetes convention for labeling. `alpha` just means the API is not stable and may change. In our case, this references the CRDs but could be features as well.